pkg/bosh/manifest/cmd_instance_group_resolver.go

```diff
@@ -113,6 +113,58 @@ func (dg *InstanceGroupResolver) Manifest() (Manifest, error) {
 	return igManifest, nil
 }
 
+// CollectQuarksLinks collect all links from path
+func (dg *InstanceGroupResolver) CollectQuarksLinks(linksPath string) error {
+	links, err := ioutil.ReadDir(linksPath)
```

Comment:
This should use `afero.Fs` at some point, so we can test it

Reply:
I changed my code. But we still had some io ops from ioutil in this package.